README.md

```diff
@@ -1,32 +1,239 @@
 ### [README still WIP, not contribution friendly yet]
 
-### Monitoring Server Configuration
-In `prometheus/prometheus.yml`, add hosts emitting metrics to the targets under scrape config's node job.
+Nodies Monitoring is a customizable and extensible monitoring solution for monitoring host machine metrics, container metrics, node status and node logs. 
+
+## Table of content
+
+- [Installation](#installation)
+    - [Docker Dependencies](#docker-dependencies)
+    - [Nodies Monitoring](#nodies-monitoring)
+- [Setup](#setup)
+    - [.env](#env)
+    - [chains.json](#chainsjson)
+    - [Promtail Logs](#promtail-logs)
+    - [Yaml Setup and Startup](#yaml-setup-and-startup)
+- [Customization](#customization)
+    - [Datasources](#datasources)
+    - [Dashboards](#dashboards)
+    - [Alerts](#alerts)
+- [License](#license)
+- [Links/Contact](#linkscontact)
+
+## Installation
+
+**Note:** Tested and recommended installation on Ubuntu 22.04.1 LTS host OS
+
+### Docker Dependencies
+<details>
+<summary>Optional section for fresh install</summary>
+
+Uninstall existing docker
+```bash
+sudo apt-get remove docker docker-engine docker.io containerd runc
+```
+
+Install required packages
+```bash
+sudo apt-get update
+sudo apt-get install \
+    ca-certificates \
+    curl \
+    gnupg \
+    lsb-release
+```
 
+Add docker official GPG key
+```bash
+sudo mkdir -p /etc/apt/keyrings
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+```
 
-### Start Monitoring Server
-Run `docker compose up` in root folder.
+Setup docker repo
+```bash
+echo \
+  "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+  $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+```
 
+Update apt package index
+```bash
+sudo apt-get update
+```
+
+Install latest docker
+```bash
+sudo apt-get install docker-ce docker-ce-cli containerd.io docker-compose-plugin
+```
+</details>
 
-### Clients/Exporters Configuration
+### Nodies Monitoring
 
-- Promtail
+Git pulldown nodies monitoring project
+```bash
+git clone --branch main https://github.com/baaspoolsllc/nodies_monitoring.git
+```
 
+Install python3-pip
+```bash
+apt install python3-pip
+```
 
-In `clients/docker-compose.yml` under the promtail service configuration, edit `- ./log:/var/log` to the location of your nginx logs.
+Install requirements
+```bash
+pip3 install -r requirements.txt
+```
 
+## Setup
+
+Our project is split into a server(monitoring) stack, and a client(exporter) stack.
```

Comment:
Can we break down the server stack

and client stack? This will lay context on the .env.

Let's include an architecture diagram on how this whole stack is works. I think it would help

Reply:
What does server do?

What does client do?